### DISCUSSION TITLE
Jazzy Jalisco Tutorial Testing party questions

### DISCUSSION THREAD

**Discussion by clalancette** [osrf/ros2_test_cases#1545] [General]
If you have questions about a particular tutorial, please leave comments in the individual issue; the admins will see them and respond.

If you have questions about the tutorial party itself, please leave a question here and the admins will field it here.

**Comment by ChidiebereMicah** [comment]
Please, i would like to know what the difference between the binary install and Debian install is. This is because, in the installation tutorial, I see that the Debian install is under binary installation (the other option being fat) but they are both listed as different options for setup in the Jazzy test readme

**Reply by clalancette** [reply]
Yes, this is a good question.

The "Debian install" is installation of Debian packages.  That is, you would do `apt-get install ros-jazzy-desktop` (or similar), which would install the packages.

The "binary install" is installation via the binary tarballs that we released in https://github.com/ros2/ros2/releases/tag/release-jazzy-beta-20240430 .  That is, you would download the tarball, uncompress it, and then run the tests using that.

Please let me know if this answers your question.

**Reply by ChidiebereMicah** [reply]
Yes, thanks. This gives more clarity.

This means the install on the normal installation tutorial page is referring to the "Debian install."

**Comment by martincerven** [comment]
Why are issues/tickets for arm64 closed? I wanted to test it on Raspberry 5 🥲

**Reply by clalancette** [reply]
They shouldn't be: see https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label:%22DDS%3A+FastDDS%22+label:%22Build+type%3A+Debian%22+label:%22Chip+set%3A+ARM64%22+label:%22OS%3A+Ubuntu+Noble+24.04%22 , for instance.

**Reply by martincerven** [reply]
I see, there is [another label](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+label%3Aarm64+is%3Aclosed) that's why I got confused.

**Comment by Shoeb2004** [comment]
Hello everyone! I am brand new to this field and this is my first time contributing to an open course. I am feeling a bit overwhelmed and unsure of what to do or how to do it. If anyone could offer some guidance, I would greatly appreciate it. I am eager to learn and excited to be a part of this community!
